### DISCUSSION TITLE
10/24 생각 나누기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#17] [동글동글 3기 피드백]
[https://yozm.wishket.com/magazine/detail/2134/](url)

글을 읽고 드셨던 생각을 간단하게 나눠주세요!

**Comment by Orm-camp** [comment]
글을 읽을 때 필요한 사전지식을 미리 언급하면, 읽는 사람 입장에서 진입장벽 (허들)이 낮아질 수 있다는 댓글이 인상적이었어요.
저도 앞으로 글을 쓸 때에는 글에서 다루는 사전지식을 앞부분에서 언급해 봐야겠다는 생각이 들었습니다!

**Comment by Devkdy22** [comment]
구구절절 길게 쓰기보다는 요약해서 중점적인 내용들이 작성되게하는게 좋겠다 라는 생각과, 길어지는 경우에는 해당 내용을 공부한 링크들을 걸어두는 방법을 사용하는게 좋을 것 같다는 생각이 들었습니다.

**Comment by hansol-woff** [comment]
과제하면서도 검색해보고 많은 페이지들을 봤는데, 단순히 정보 전달성 글을 넘어서서 흥미로운 컨텐츠로 보이게 하는게 가장 어려울 것 같다는 생각이 들었습니다.

**Comment by hyebinkimsdf** [comment]
포스팅은 큰 내용보다는 자세한 상황을 예를 들어 작성하는게 좋다는 점에서 알아갔습니다.

**Comment by hyeoholee** [comment]
개발자들의 생각들이 많이 보여서 같은 문제를 겪었을 때 찾아서 풀어나가기도 보기 좋고
개발자에게 질문, 댓글들도 달아서 같은 고민의 문제에 해답 찾는 것이 빠를 거 같습니다.

**Comment by baaanjy** [comment]
동기부여가 되는 내용이에요. 단순히 공식문서 내용을 그대로 쓰는 것보다는 개인의 생각과 경험을 잘 녹여서 쓰는게 중요할 것 같아요.

**Comment by oratio100** [comment]
눈에 들어오는 블로그를 만들어야 효과가 있을 것 같은데
주제에 맞는 글을 적당한 분량으로 쓰는 방법을 익히는데 시간이 좀 걸릴 것 같다고 생각했습니다.

**Comment by mh0223** [comment]
기술 블로그는 정보전달보다는 경험 공유가 중점이 되어야할 것 같습니다. 블로그 작성 전에 해당 정보와 관련해서 많이 구현해보고 시도해봐야할 것 같습니다!

**Comment by ksj0621** [comment]
너무 많은 정보를 전달하려 하기 보다 참고링크를 달아서 글을 깔끔하고 주제의식이 명확히 드러나도록 작성하는 것이 좋은 방법이란 것을 알게되었습니다.

**Comment by ghost** [comment]
교육 받은 내용을 토대로 
본인의 블로그를 만듦으로써 좀 더 본인 것 이라는 애착에 자발적으로 찾아보며 꾸밀 것 같다 .

라는 생각이 들었습니다.

**Comment by ghost** [comment]
기술 블로그라고 들었을 때 전문적인 내용 위주로 작성하는지 알았는데 문제해결 과정과 경험이 중요하다고 느꼈습니다.

**Comment by structified** [comment]
정말 열심히 사는 분들이 많다고 새삼 느끼게 되었습니다.
내가 놓치면서 살고 있는 부분이 있지는 않은지 다시 한번 생각해 보게 됩니다.

**Comment by kimjigyun** [comment]
많은 블로그들을 찾아보았는데, 주제에 맞는 표현들을 적절히 써야겠고 / 글 뿐만 아니라 이해를 위해 보기 좋게 이미지를 섞어도 좋겠다 싶었습니다.  마지막에 시도해보라고 하니까 저도 시간 쪼개서 기술 정리나 경험들을 시도해보면 나중에 너무 좋을듯합니다.
